### ISSUE TITLE
About Loss Function

### ISSUE THREAD

**Issue by ssdutHB** [moabitcoin/holy-edge#13]
Hi, everyone. I read the code in data_parser.py, found that there is an option for input labels, that is "target regression". If we choose this option, the loaded ground truth will be a matrix consisting of real numbers from 0 to 1 rather than a binary matrix. After that, I checked losses.py and found that there are two lines of codes:
"count_neg = tf.reduce_sum(1. - y)"
"count_pos = tf.reduce_sum(y)"

there two lines of codes seems work well for a binary ground truth, but if they work for a ground truth that consists of real numbers from 0 to 1? I am looking forward to your answers.
